src/nisarqa/raster_classes.py

```diff
@@ -635,10 +685,10 @@ class GeoGrid(CoordinateGrid):
 
     epsg: int
 
-    x_spacing: float
+    x_axis_posting: float
```

Comment:
```suggestion
    # Call this `x_axis_posting` instead of simply `x_posting` to avoid
    # name collisions with the property in the base class.
    x_axis_posting: float
```